src/tike/ptycho/probe.py

```diff
@@ -871,6 +895,35 @@ def finite_probe_support(probe, *, radius=0.5, degree=5, p=1.0):
     return p * mask.astype(tike.precision.floating)
 
 
+def rescale_probe_using_fixed_intensity_photons(
+    probe,
+    Nphotons,
+    probe_occ=None,
+):
+    """
+    Rescales the shared probe modes so that its intensity L2 norm equals to some number of photons.
+
+    Parameters
+    ----------
+    Nphotons : float (0, inf)
+        The total number of photons in the L2 norm of the shared probe mode intensity
+
+    probe_occ :
```

Comment:
What does OCC stand for? It's not obvious to me, so the name of this param should be changed or the explanation should be included in the docstring.

Reply:
Fixed I think?

Reply:
This updated parameter name for probe_occ is much better. Thanks!